### DISCUSSION TITLE
Spotify features

### DISCUSSION THREAD

**Discussion by i120ms** [KRTirtho/spotube#1014] [General]
I have a question about stuff like wrapped, time spent and overall statistics that Spotify has. 
Is this app has any configured with statistics from Spotify?
I will be really thankful for answer :>

PS: It is really good optimized.

![image](https://github.com/KRTirtho/spotube/assets/151391534/cb534e22-ca89-4775-aae4-43540feaf971)

**Comment by KRTirtho** [comment]
No, nothing is sent to Spotify.
